recipes/napari-tiler/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "napari-tiler" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-tiler-{{ version }}.tar.gz
+  sha256: d13096502205fb4c5e9ca54656b350ec3e2b7c104162910d45f9bb6f0e09a78d
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry-core
+    - python
+  run:
+    - importlib-metadata <4.3  # [py<38]
```

Comment:
This could be `noarch` but since this requirement pins an old version, we don't have the noop packages for py 3.10. So I guess we will leave as non-noarch :(